client/js/contextMenuFactory.js

```diff
@@ -125,6 +125,74 @@ function addKickItem() {
 	});
 }
 
+function addOpItem() {
+	function op(itemData) {
+		socket.emit("input", {
+			target: $("#chat").data("id"),
+			text: "/op " + itemData,
+		});
+	}
+
+	addContextMenuItem({
+		check: (target) => utils.hasRoleInChannel(target.closest(".chan"), ["op"]) && utils.hasRoleInChannel(target.closest(".chan"), ["op"], target.data("name")) === false,
```

Comment:
Can you break this line and others below?

I'll post a diff, probably simpler for you.